adoc/chapters/glossary.adoc

```diff
@@ -432,14 +432,16 @@ object. For the full description please refer to <<subsec:buffers>>.
     be inlined, public and defaulted, none of them should be explicitly
     declared.
 
-[[smcp]]SMCP::
-    The single-source multiple compiler-passes (SMCP) technique allows a
-    single source file to be parsed by multiple compilers for building
-    native programs per compilation target. For example, a standard {cpp} CPU
-    compiler for targeting <<host>> will parse the <<sycl-file>> to
-    create the {cpp} <<sycl-application>> which offloads parts of the
-    computation to other <<device,devices>>. A SYCL device compiler will parse
-    the same source file and target only SYCL kernels.
+[[smcp]]SMCP:: The single-source multiple compiler-passes (SMCP)
+    technique allows a single-source file to be parsed by multiple
+    compilers for building native programs per compilation target. For
+    example, a standard {cpp} CPU compiler for targeting <<host>> will
+    parse the <<sycl-file>> to create the {cpp} <<sycl-application>>
+    which offloads parts of the computation to other
+    <<device,devices>>. A SYCL device compiler will parse the same
+    source file and target only SYCL kernels. For the full description
+    please refer to <<subsec:smcp>>.  See <<sscp>> for the dual
+    approach.
```

Comment:
I'm not sure about the last sentence.  Should we say:

> See \<\<sscp\>\> for another approach.

?

Reply:
You are right. Let's not use pseudo-mathematical jargon here.